shell/packages/sandstorm-db/db.js

```diff
@@ -492,6 +500,15 @@ ApiTokens = new Mongo.Collection("apiTokens", collectionOptions);
 //         address :Text;
 //       }
 //       identity :Text;
+//       http :group {
+//         url :Text;
+//         auth :union {
+//           none :Void;
+//           bearer :Text;
+//           basic :group { username :Text; password :Text; }
```

Comment:
We could encrypt them using the ApiToken as a key, since we only store the hash of the API token in the database. Child ApiTokens would store an encrypted copy of the parent token (in addition to the parent token ID as they do today). But a complete solution would require that app storage be encrypted as well, since that's where the tokens are ultimately stored. That is, of course, a big project...